content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -886,7 +886,7 @@ enum Role {
 
 <Admonition type="info">
 
-Composite types are available in Preview in versions `3.10.0` and later.
+Composite types were added in version `3.10.0` under the `mongodb` Preview feature flag and have been Generally Available since version `3.12.0`.
```

Comment:
```suggestion
Composite types were added in version `3.10.0` under the `mongodb` Preview feature flag and are in General Availability since version `3.12.0`.
```